packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -35,10 +52,17 @@ export class CoinbaseWalletProvider extends ProviderEventEmitter implements Prov
       preference,
     });
 
+    if (this.preference.enableAutomaticSubAccountCreation) {
```

Comment:
@montycheese which 3rd party lib was this aiming to fix? I think we can move this up into the CoinbaseWalletSDK class vs the provider but want to make sure I test that edge case

Reply:
it was wagmi. 

it was just a way to set the default toSubAcccountSigner impl if one wasn't being provided, since wagmi doesn't currently see all of our sdk params, just the preferences map.